apps/dapp-console-api/src/routes/faucet/FaucetRoute.spec.ts

```diff
@@ -0,0 +1,221 @@
+import {
+  createSignedInCaller,
+  mockDB,
+  mockLogger,
+  mockPrivyClient,
+  validSession,
+} from '@/testhelpers'
+
+describe(FaucetRoute.name, () => {
+  let publicClient: PublicClient
+  let walletClient: WalletClient<Transport, Chain, Account>
+  let testClient: TestClient
+  let faucetAddress: Address
+  let onChainFamAddress: Address
+  let offChainFamAddress: Address
+  let faucetAdminAccount: PrivateKeyAccount
+  let ownerAccount: PrivateKeyAccount
+  let recipientAccount: PrivateKeyAccount
+
+  let trpc: Trpc
+  let caller: ReturnType<RouterCaller<FaucetRoute['handler']['_def']>>
+  const redisCache = new RedisCache('redis://redis-app:6739')
+
+  beforeEach(async () => {
+    const session = await validSession()
+    const privyClient = mockPrivyClient()
+
+    trpc = new Trpc(privyClient, mockLogger, mockDB)
+
+    publicClient = createL2PublicClient()
+    walletClient = createL2WalletClient()
+    testClient = createL2TestClient()
+
+    faucetAdminAccount = privateKeyToAccount(
+      '0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80',
+    )
+    ownerAccount = privateKeyToAccount(
+      '0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d',
+    )
+    recipientAccount = privateKeyToAccount(
+      '0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a',
+    )
+
+    faucetAddress = await deployFaucetContract(
+      publicClient,
+      walletClient,
+      faucetAdminAccount,
+    )
+
+    await testClient.setBalance({
+      address: faucetAddress,
+      value: parseEther('1000'),
+    })
+
+    onChainFamAddress = await deployOnChainFamContract(
+      publicClient,
+      walletClient,
+      faucetAdminAccount,
+    )
+
+    offChainFamAddress = await deployOffChainFamContract(
+      publicClient,
+      walletClient,
+      faucetAdminAccount,
+    )
+
+    const oneDayInSeconds = 1 * 60 * 60 * 24
+
+    await walletClient.writeContract({
+      abi: faucetAbi,
+      address: faucetAddress,
+      functionName: 'configure',
+      args: [
+        onChainFamAddress,
+        {
+          ttl: BigInt(oneDayInSeconds),
+          amount: parseEther('1'),
+          name: 'ON_CHAIN_ADMIN_FAM',
+          enabled: true,
+        },
+      ],
+      account: faucetAdminAccount,
+      chain: walletClient.chain,
+    })
+
+    await walletClient.writeContract({
+      abi: faucetAbi,
+      address: faucetAddress,
+      functionName: 'configure',
+      args: [
+        offChainFamAddress,
+        {
+          ttl: BigInt(oneDayInSeconds),
+          amount: parseEther('.05'),
+          name: 'OFF_CHAIN_ADMIN_FAM',
+          enabled: true,
+        },
+      ],
+      account: faucetAdminAccount,
+      chain: walletClient.chain,
+    })
+
+    const route = new FaucetRoute(trpc, [
+      new Faucet({
+        chainId: 11155111,
+        redisCache: redisCache,
+        faucetAddress: faucetAddress,
+        displayName: 'OP Sepolia',
```

Comment:
This should be Sepolia since 11155111 is Sepolia. Also we should do this on l1 so that the tests do not involve the bridging logic